src/utils/clmfates_interfaceMod.F90

```diff
@@ -583,6 +580,8 @@ subroutine init(this, bounds_proc )
                end if
             endif
 
+            col%copy_fates_var(c) = .false.
```

Comment:
@ekluzek I ended up using col%is_fates(c) as a template to get copy_fates_var to work in the threaded test.

You recommended "staying local" if possible, but none of my corresponding experiments succeeded:
- I tried allocating locally in this subroutine init with
`copy_fates_init(bounds_clump%begc:bounds_clump%endc)`
but failed with error "trying to allocate an already allocated variable."
- I tried allocating locally in subroutine Init2, and the continuous run finished, but the restart run failed with a segfault. This may mean that restarts don't run through subroutine Init2.

If you have another suggestion that you would like me to try, pls let me know, though I probably won't get to it before Monday. In the mean time, I think I'll update to the latest tag and start the test suites, so as to be ready by Monday if no tests fail :-)

Reply:
> In the mean time, I think I'll update to the latest tag and start the test suites

cheyenne Ok
izumi Ok
...expected failures + FIELDLIST field lists differ (otherwise bit-for-bit)